test/unit/role_test.lua

```diff
@@ -487,6 +519,93 @@ function g.test_apply_config_start_tasks(cg)
     t.assert_not_equals(cg.role.task(task_name2), nil)
 end
 
+function g.test_apply_config_cfg_metrics_default(cg)
+    t.skip_if(not helpers.is_metrics_supported(),
+              "metrics >= 0.11.0 is not installed")
+    local task_name1 = "apply_config_test1"
+    local metrics_default = expirationd.cfg.metrics
+
+    cg.role.apply_config({
+        expirationd = {
+            [task_name1] = {
+                space = g.space.id,
+                is_expired = always_true_func_name,
+            },
+        }
+    }, {is_master = false})
+
+    t.assert_equals(#cg.role.tasks(), 1)
+    t.assert_not_equals(cg.role.task(task_name1), nil)
+    t.assert_equals(expirationd.cfg.metrics, metrics_default)
+
+    expirationd.cfg({metrics = metrics_default})
```

Comment:
Shouldn't it be in `after_test`/`after_each` hook?

Reply:
Thank you, updated.